ginga/util/iqcalc.py

```diff
@@ -23,13 +23,32 @@
 
 
 def get_mean(data_np):
-    mdata = np.ma.masked_array(data_np, np.isnan(data_np))
```

Comment:
Note: Assumption is that `data_np` is *always* a Numpy array. If you want to pass in list, there needs to be extra processing.

Reply:
Hmm...it's on my TODO list to support `dask` arrays.

Reply:
If `dask` array quacks like Numpy, this should be okay? I'm not familiar enough to say for sure.

Reply:
Apparently `dask` supports some `numpy` ufuncs, but not all.  It also only supports a subset of the `numpy` fancy indexing, some of which Ginga uses.

Reply:
I am definitely open to modifying these for `dask` integration in a future PR. 😄